README.md

```diff
@@ -1,37 +1,34 @@
 # DocumenterCitations.jl
-Using [Bibliography.jl](https://github.com/Azzaare/Bibliography.jl) to add support for BibTeX citations and references in documentation pages generated by [Documenter.jl](https://github.com/JuliaDocs/Documenter.jl).
+DocumenterCitations.jl uses [Bibliography.jl](https://github.com/Azzaare/Bibliography.jl) to add support for BibTeX citations and references in documentation pages generated by [Documenter.jl](https://github.com/JuliaDocs/Documenter.jl).
 
 DocumenterCitations.jl is still in early development so please open issues if you encounter any bugs or pain points, would like
 to see a new feature, or if you have any questions.
 
-## Installation instructions
-
-You can install the latest version of DocumenterCitations.jl using the built-in package manager (accessed by pressing `]` in the
-Julia command prompt) to add the package and instantiate/build all dependencies
-
-```julia
-julia>]
-(v1.5) pkg> add DocumenterCitations
-```
-
-## Quickstart
-
-You just need a BibTeX `.bib` file to get started! Let's call it `some_references.bib`. You want to create a `CitationBibliography`
-and pass it to `makedocs`:
-
-```julia
-using DocumenterCitations
-
-cite_bib = CitationBibliography("some_references.bib")
-makedocs(cite_bib, ...)
-```
-
-Now you can use
-
-```
-    ```@bibliography
-    ```
-```
-
-on any page to generate a full list of formatted references. You can also use `[key](@cite)` on any page where `key` is the name of
-the BibTex entry to create a citation that automatically links to the appropriate entry in the bibliography.
+<!-- Information badges -->
+<p align="center">
+  <a href="https://www.repostatus.org/#active">
+    <img alt="Repo status" src="https://www.repostatus.org/badges/latest/active.svg?style=flat-square" />
+  </a>
+  <a href="https://mit-license.org">
+    <img alt="MIT license" src="https://img.shields.io/badge/License-MIT-blue.svg?style=flat-square">
+  </a>
+  <a href="https://github.com/ali-ramadhan/DocumenterCitations.jl/issues/new">
+    <img alt="Ask us anything" src="https://img.shields.io/badge/Ask%20us-anything-1abc9c.svg?style=flat-square">
+  </a>
+  <a href="https://github.com/ali-ramadhan/DocumenterCitations.jl/releases">
+    <img alt="GitHub tag (latest SemVer pre-release)" src="https://img.shields.io/github/v/tag/ali-ramadhan/DocumenterCitations.jl?include_prereleases&label=latest%20version&logo=github&sort=semver&style=flat-square">
+  </a>
+</p>
+
+<!-- CI and documentation badges -->
+<p align="center">
+  <a href="https://travis-ci.com/ali-ramadhan/DocumenterCitations.jl">
+    <img alt="CPU build status" src="https://img.shields.io/travis/com/ali-ramadhan/DocumenterCitations.jl/master?label=Travis&logo=travis&logoColor=white&style=flat-square">
+  </a>
+  <a href="https://ali-ramadhan.github.io/DocumenterCitations.jl/stable">
```

Comment:
```suggestion
  <a href="https://ali-ramadhan.github.io/DocumenterCitations.jl/dev">
```
I suspect the image line below needs to be changed also

Reply:
Ah good catch!

Reply:
Ah wait I think the link was correct but lead to a non-existing page as we haven't tagged any release (with docs) yet. So "stable documentation" won't exist until we tag the next version, but then the link should be correct.

Reply:
Idk that `stable` is automatically made by default anymore (see [CLIMAParameters dev](https://clima.github.io/CLIMAParameters.jl/dev/) [CLIMAParameters stable](https://clima.github.io/CLIMAParameters.jl/stable/) ). We might need to modify something in Documenter to get stable docs.

Reply:
Hmm, the [Documenter.jl docs](https://juliadocs.github.io/Documenter.jl/stable/man/hosting/#Documentation-Versions) claim they are made 🤷🏻‍♂️

Reply:
That doesn't explain my comment about CLIMAParameters, though..

Reply:
Hmmm yeah CLIMAParameters.jl seems to be deploying to `dev` even on tags. Couldn't find any reason why it would do that though...

Reply:
I'm fine trying it this way, but it seems to me that we'd risk having a tagged release without a link to docs.

Reply:
Ah that's a good point. Committed your suggestion again so that we don't end up with any broken links. And we can fix it when we tag a release.